### ISSUE TITLE
Code organization / refactoring

### ISSUE THREAD

**Issue by goanpeca** [spyder-ide/spyder#9035]
- [ ] Move preferences to app folder
- [ ] Move all static assets to an assets or static folder

**Comment by ccordoba12** [comment]
I don't agree with moving preferences. They are independent from the main window and deserve a category on its own.
